content/pt/docs/guias/WEBHOOKS/_index.md

```diff
@@ -1,22 +1,26 @@
 ---
-title: "Webhooks"
-slug: "webhooks-guides"
-hidden: false
-date: "2019-07-29T21:31:30.337Z"
-lastmod: "2021-02-01T22:57:33.698Z"
-weight: 7
+title: "WEBHOOKS"
+linkTitle: "WEBHOOKS"
```

Comment:
links should be lower case to follow the standard

Reply:
Let's follow with this new pattern, I changed it in references too. The main menu will be in capital letters and the submenus will be in lower case.